DOC-6363: 4.11.3 patch release (#2073)
corda/corda-docs-portal: content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

If you are using the Archive Service with Corda Enterprise Edition 4.11, you must use the 1.1.x stream of the Archive Service release. For more details, see [Archive Service]({{< relref "../../../../tools/archiving-service/archiving-release-notes.md" >}}).
{{< /note >}}

## Corda Enterprise Edition 4.11.3 release notes

Corda Enterprise Edition 4.11.3 is a patch release of Corda Enterprise Edition focused on resolving issues.

### Upgrade recommendation

As a developer or node operator, you should upgrade to the [latest released version of Corda]({{< relref "../enterprise/_index.md" >}}) as soon as possible. The latest Corda Enterprise release notes are on this page, and for the latest upgrade guide, refer to [Upgrading a CorDapp or node](upgrading-index.md).

### Fixed issues

> * When executing the database migration, it was possible for the upgrade to fail when upgrading from a pre-4.11 node. This occurred when the database contained new-style confidential identities, that is, those without a certificate. This issue has now been resolved.

Reviewer: ```suggestion
* Database upgrades failed when upgrading from a pre-4.11 node, if the database contained new-style confidential identities, that is, those without a certificate. 
```
